### PR TITLE
UI updates

### PR DESCRIPTION
**Pull request by jessebmiller**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "./react/node_modules/typescript/lib"
```
Comment (jessebmiller) [inline .vscode/settings.json:2]: You should be able to set a personal global .gitignore to ignore this file

Comment (jessebmiller) [reply .vscode/settings.json:2]: not worried about it now though

File: scripts/examples/output/1640178091526.txt [hunk 2]
```diff
@@ -0,0 +1,387 @@
+STORY 0-0
```
Comment (jessebmiller) [inline scripts/examples/output/1640178091526.txt]: We probably want to git ignore these

**Review by jessebmiller — COMMENTED** [review]
I'm going to remove and ignore the example output then I'm good to merge this

**Review by jessebmiller — COMMENTED** [review]
Looks good to me now

**Comment by EzraWeller** [comment]
All that CI stuff is failing, but other than that looks good. Is that stuff supposed to be working or are we ignoring atm?

**Pull request merged**